apps/charterafrica/src/payload/blocks/FeaturedPost.js

```diff
@@ -0,0 +1,30 @@
+const FeaturedPost = {
+  slug: "featured-post",
+  labels: {
+    singular: {
+      en: "Featured Post",
+      pt: "Postagem em destaque",
+      fr: "Après sélectionnée",
+    },
+    plural: {
+      en: "Featured Post",
+      pt: "Postagem em destaque",
+      fr: "Après sélectionnée",
+    },
+  },
+  fields: [
+    {
+      name: "featuredPost",
+      label: {
+        en: "Featured Post",
+        pt: "Postagem em destaque",
+        fr: "Après sélectionnée",
+      },
+      type: "relationship",
+      relationTo: ["news"],
+      required: true,
```

Comment:
Have you checked if we can have multiple `relationTo` and the data structure wouldn't change when we add those? Remember, once data is entered by the content team, it will be very difficult to change its structure afterwards.

Reply:
@kilemensi Here, [docs](https://payloadcms.com/docs/fields/relationship) explain how data is saved for dynamic references.

Reply:
I know about the docs @koechkevin ... I wanted **you** to go through it and arrive at either  **Has One - Polymorphic** if we're only ever going to have one featured post or **Has Many - Polymorphic** if we're allowing multiple featured docs

I think we're missing `hasMany`.

Reply:
Got it